usaspending_api/disaster/v2/views/agency/count.py

```diff
@@ -19,25 +19,24 @@ class AgencyCountViewSet(AwardTypeMixin, CountBase):
 
     @cache_response()
     def post(self, request: Request) -> Response:
+        filters = [
+            self.all_closed_defc_submissions,
+            self.has_award_of_provided_type,
+            self.is_in_provided_def_codes,
+        ]
+
         if self.award_type_codes:
-            filters = [
-                self.is_in_provided_def_codes(),
-                self.all_closed_defc_submissions,
-                self.is_non_zero_award_spending(),
-            ]
+            filters.append(self.is_non_zero_award_spending)
```

Comment:
minor, but this can probably move to before the if statement, since this filter is also added in the else condition.

Reply:
Look closer at the property names

Reply:
I see now. I have been a fool.